src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import {
+  BrowserTrackConfig,
+  BrowserTrackConfigProps
+} from './BrowserTrackConfig';
+import Checkbox from 'src/shared/components/checkbox/Checkbox';
+
+import {
+  createTrackConfigLabel,
+  createTrackConfigNames,
+  createCogTrackList
+} from 'tests/fixtures/browser';
+
+describe('<BrowserTrackConfig />', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const defaultProps: BrowserTrackConfigProps = {
+    applyToAll: false,
+    browserCogTrackList: createCogTrackList(),
+    selectedCog: 'track:gc',
+    trackConfigLabel: createTrackConfigLabel(),
+    trackConfigNames: createTrackConfigNames(),
+    updateApplyToAll: jest.fn(),
+    updateTrackConfigLabel: jest.fn(),
+    updateTrackConfigNames: jest.fn(),
+    onClose: jest.fn()
+  };
+
+  let wrapper: any;
+
+  beforeEach(() => {
+    wrapper = mount(<BrowserTrackConfig {...defaultProps} />);
+  });
+
+  describe('rendering', () => {
+    test('renders the all tracks checkbox', () => {
+      expect(wrapper.find(Checkbox)).toHaveLength(1);
+    });
+
+    test('renders all the buttons', () => {
+      expect(wrapper.find('button')).toHaveLength(6);
+    });
+  });
```

Comment:
these two tests aren't really useful. They are (partially?) covered in the behaviour section. Let's remove them.